### PR TITLE
modify `--debug` to be without brk and add `--debug-brk`.

### PR DESCRIPTION
**Pull request by luxferresum**
This adresses #429.

Maybe this is a breaking change because its actually changing the cli API? I'm not sure about this.

### CODE REVIEW

File: commands/server.ts [hunk 1]
```diff
@@ -147,9 +152,13 @@ export default class ServerCommand extends Command {
   protected startServer(argv: any, project: Project) {
     let bootstrapPath = project.isAddon ? path.join('test/dummy/index.js') : 'index.js';
     let args = [ bootstrapPath ];
-    if (argv.debug) {
+    if (argv.debugBrk) {
       args.unshift('--inspect-brk');
     }
+    if(argv.debug) {
```
Comment (knownasilya) [inline commands/server.ts]: Would we ever want both of these flags applied?

Comment (luxferresum) [reply commands/server.ts]: I dont think its a problem. `--inspect --inspect-brk` is equivalent AFAIK. No reason to add a check here.

File: commands/server.ts [hunk 2]
```diff
@@ -42,7 +42,12 @@ export default class ServerCommand extends Command {
       type: <any>'string'
     },
     debug: {
-      description: 'Run in debug mode (add the --debug flag to node, launch node-inspector)',
+      description: 'Run in debug mode (add the --inspect flag to node, launch node-inspector)',
```
Comment (davewasmer) [inline commands/server.ts]: We should drop the `node-inspector` comment, since that's no longer the actual behavior.

File: commands/server.ts [hunk 3]
```diff
@@ -42,7 +42,12 @@ export default class ServerCommand extends Command {
...
+      default: false,
+      type: <any>'boolean'
+    },
+    debugBrk: {
+      description: 'Run in debug mode (add the --inspect-brk flag to node, launch node-inspector)',
```
Comment (davewasmer) [inline commands/server.ts]: Ditto here

**Comment by luxferresum** [comment]
Fixed the requested changes.

**Comment by luxferresum** [comment]
whats the status here?

**Pull request merged**